provision/opentofu/modules/aws/accelerator/src/stonith_lambda.py

```diff
@@ -40,6 +53,9 @@ def handle_site_offline(labels):
             EndpointConfigurations=endpoints
         )
         print(f"Removed site={offline_site} from Accelerator EndpointGroup")
+
```

Comment:
not related to this line...
where is the label `site` defined? In line 48, you have `offline_site = labels['site']` but in `infinispan-alerts.yaml` no label with the name `site` exists.

Reply:
The label is automatically included because we use it in the metric query, the same is true of `namespace`